src/App.js

```diff
@@ -16,6 +16,7 @@ function App() {
       <Nav />
       <Routes>
         <Route path="/" element={<Home />} />
+        <Route path={`/articles`} element={<Home />} />
```

Comment:
Assuming you added this for when you Link to `/articles?topic=topic`. It'd still work on your home page so you can keep it on the same path every time. 

Up to you if you wanna keep it here to make it clearer though 👍